azure/packages/azure-client/package.json

```diff
@@ -29,13 +29,12 @@
 	"scripts": {
 		"api": "fluid-build . --task api",
 		"api-extractor:commonjs": "api-extractor run --local",
-		"api-extractor:esnext": "copyfiles -u 1 \"dist/**/*-@(alpha|beta|public|untrimmed).d.ts\" lib",
+		"api-extractor:esnext": "api-extractor run --config ./api-extractor-esm.json",
```

Comment:
Should all of these also have `--local`?
Or should none of `:commonjs` have `--local`?

Reply:
One of them needs to generate the markdown report file, and that's the commonjs one, so it has --local so the file is generated during local builds. In ci the ci:build:docs task is run, which omits --local.

I don't think there's any difference for the ESM one since it's not configured to generate a report file.

Reply:
I see. I missed the part where there is an update of *.api.md with `--local` and not without.
I think that means we need to pick one or the other of `api-extractor:*` to run first and make the second dependent on the first -- second should verify that *.api.md files are correct (and same for both). I would pick `:commonjs` to be first. Where there is no `:commonjs` then `:esnext` should have `--local`. Sound right?

Reply:
> I would pick :commonjs to be first. Where there is no :commonjs then :esnext should have --local. Sound right?

Yeah, sounds right to me.